lib/bibliothecary/parsers/npm.rb

```diff
@@ -74,8 +74,12 @@ def self.parse_package_lock_deps_recursively(dependencies, depth=1)
       def self.parse_manifest(file_contents, options: {})
         manifest = JSON.parse(file_contents)
         raise "appears to be a lockfile rather than manifest format" if manifest.key?('lockfileVersion')
-        map_dependencies(manifest, 'dependencies', 'runtime') +
-        map_dependencies(manifest, 'devDependencies', 'development')
+        
+        (
+          map_dependencies(manifest, 'dependencies', 'runtime') +
+          map_dependencies(manifest, 'devDependencies', 'development')
+        )
+          .reject { |dep| dep[:name] == "//" } # Omit comment keys. They are valid in package.json: https://groups.google.com/g/nodejs/c/NmL7jdeuw0M/m/yTqI05DRQrIJ
```

Comment:
Is npm open source? Might be handy to find the source for this, and confirm there's no other special cases for it. Like whitespace on either side of the "//" for example.

Reply:
I tried to, and can't find anything 😆 I couldn't find anything in node either, but I'm going to check again